vng_api_common/views.py

```diff
@@ -172,8 +181,7 @@ def _test_nrc_config() -> list:
     from notifications_api_common.models import NotificationsConfig, Subscription
 
     nrc_config = NotificationsConfig.get_solo()
-
-    nrc_client = NotificationsConfig.get_client()
+    nrc_client: Optional[Client] = NotificationsConfig.get_client()
```

Comment:
I think on the line 178 `notifications_api_common` app should be checked

Reply:
good catch! I'll change it